ui/desktop/app/routes/scopes/scope/projects/targets/target.js

```diff
@@ -86,6 +85,24 @@ export default class ScopesScopeProjectsTargetsTargetRoute extends Route {
     return true;
   }
 
+  /**
+   * Removes any errors from target model when transitioning away
+   * from this route and sets isConnectionError to false if true.
+   * @returns {boolean}
+   */
+  @action
+  willTransition() {
```

Comment:
Is the proper event `willTransition` or `resetController`? It seems like it's more of the latter?

Reply:
I think `resetController` makes more sense [as per documentation](https://guides.emberjs.com/v4.12.0/routing/query-params/), but do we also set `this.isConnectionError` with the transition, as per [previous comment](https://github.com/hashicorp/boundary-ui/pull/1960/files#r1376410464) we might not need it. Then it makes even more sense the usage of `resetController`